app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,17 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(self, nickname: str,
+                 favourite_dish: str,
+                 hummer_level: int) -> None:
```

Comment:
There is a typo in the parameter name `hummer_level`. It should likely be `hammer_level` to reflect the intended meaning. This change should be applied consistently throughout the class.